assets/app/scripts/directives/util.js

```diff
@@ -169,7 +169,7 @@ angular.module('openshiftConsole')
   }).directive('onEnter', function(){
     return function (scope, element, attrs) {
       element.bind("keydown keypress", function (event) {
-        if(event.which === 13 && !scope.form.$invalid) {
```

Comment:
assuming you double checked everything that was using this directive to make sure it appropriately validates?  don't we have forms where the submit button was disabled until certain required fields were filled in, those forms may not expect the submit/save function to be called